### DISCUSSION TITLE
How do i edit width of the input label

### DISCUSSION THREAD

**Discussion by rajeevgondi** [wojtekmaj/react-daterange-picker#354] [Q&A] [answered]
Like i want to edit the width of the year placeholder, how can i do it ?

**Comment by Valent1nn** [comment, marked as answer]
That is simply done like this:
.react-daterange-picker__inputGroup__input.react-daterange-picker__inputGroup__year {
  width: 500px !important;
 }
 
 Adjust your width accordingly!
